virttest/utils_conn.py

```diff
@@ -2117,37 +2117,39 @@ def del_firewall_ports(self, session):
         for port_to_del in [self.desturi_port, self.migrateuri_port, self.disks_uri_port]:
             firewall_cmd.remove_port(port_to_del, 'tcp')
 
-    def install_qemu_kvm_pp(self, session, qemu_kvm_pp_path):
+    def install_qemu_kvm_pp(self, session, qemu_kvm_pp_path, timeout=240):
         """
         Install qemu-kvm module by semodule
 
         :param session: Session object
         :param qemu_kvm_pp_path: Path of qemu-kvm.pp file
+        :param timeout: Timeout to execute command lines
         """
         cmd = "semodule -l|grep qemu-kvm"
         status, output = session.cmd_status_output(cmd)
         if status:
             logging.debug("Active qemu-kvm policy.")
             cmd = "semodule -i %s" % qemu_kvm_pp_path
-            status, output = session.cmd_status_output(cmd)
+            status, output = session.cmd_status_output(cmd, timeout=timeout)
```

Comment:
a little wonder two items:
1)How can ensure for any session, its session.cmd_status_output() support timeout parameter considering session (as one input parameter)may vary
2)generally, if without specific timeout ,it should be never timeout. So by specifying 240 timeout , it sound like it may not fix the issue.

Reply:
> a little wonder two items:
> 1)How can ensure for any session, its session.cmd_status_output() support timeout parameter considering session (as one input parameter)may vary
In this case, session is aexpect.client.RemoteSession, so cmd_status_output supports timeout. 
In other codes, sorry, I have no idea, I think you have to check the code...
> 2)generally, if without specific timeout ,it should be never timeout. So by specifying 240 timeout , it sound like it may not fix the issue.
No, the default value of timeout here is 60s, and shell command line needs more than 60s on some machines. And after extending it to 240s, the problem is fixed, please check the test result above.

Reply:
For 1) Do we need considering session may not be aexpect.client.RemoteSession?
try:
  `status, output = session.cmd_status_output(cmd, timeout=timeout)
except:
   status, output = session.cmd_status_output(cmd,)

`

Reply:
No, I don't think so. 
The class, UNIXSocketConnection, is to prepare unix socket environment on a local and remote host. So the session here always support time parameter. Hence we shouldn't use try-except.

Reply:
@chunfuwen , In my POV, this is only a simple fix to enable specific timeout value for installing/uninstalling. And this is normal usage for an aexpect session. So it is ok to take it easy, I think.